docs/getting-started-guides/coreos/azure/index.md

```diff
@@ -1,246 +0,0 @@
----
```

Comment:
We may want to keep the URL, WDYT? We can point people at https://github.com/weaveworks-guides/weave-kubernetes-coreos-azure... but actually, do people still use bookmarks? If anyone release used it, they'd know where to find the repo. Personally, I don't care too much, but I've briefly touch on removal of outdated turn-up docs with some folks at the dev summit, and heard that redirects are desired. (See also https://github.com/kubernetes/kubernetes.github.io/issues/1659).